### ISSUE TITLE
[rllib] The `kl_coeff` parameter can be infinity if `kl_target` is not finetuned

### ISSUE THREAD

**Issue by yangysc** [ray-project/ray#18492]
### What is the problem?
Recently, I found that if we do not finetune the `kl_target` paramter in `ppo_torch_policy.py`, 
and the `sample_kl` is always larger than `2 * kl_target`, then the `kl_coeff` would be always multiplied by `1.5`. It would be **infinity** if `sampled_kl` is always large for some tasks. So the total loss for ppo would also be infinity.
https://github.com/ray-project/ray/blob/d314d0c10eb7677ff6638d94118a2caeba9af419/rllib/agents/ppo/ppo_torch_policy.py#L187-L194
![image](https://user-images.githubusercontent.com/8980981/132855825-7bffc1e1-923a-4b19-8774-7a194a0754f4.png)

![image](https://user-images.githubusercontent.com/8980981/132855755-a47bd98b-d26a-4e12-91a3-d8efdb76135b.png)

Though it is the user's duty to set proper parameters, maybe we should set a warning about it, or set the maximum `kl_coeff` value to help the user debug more easily?

*Ray version and other system information (Python version, TensorFlow version, OS):*

### Reproduction (REQUIRED)
Please provide a short code snippet (less than 50 lines if possible) that can be copy-pasted to reproduce the issue. The snippet should have **no external library dependencies** (i.e., use fake or mock data / environments):

If the code snippet cannot be run by itself, the issue will be closed with "needs-repro-script".

- [ ] I have verified my script runs in a clean environment and reproduces the issue.
- [ ] I have verified the issue also occurs with the [latest wheels](https://docs.ray.io/en/master/installation.html).

**Comment by xwjiang2010** [comment]
@sven1977 Maybe something to add to "RLlib documentation overhaul"?

**Comment by mvindiola1** [comment]
@sven1977 @gjoliver

Should we perhaps set the default kl_koeff to 0.0. I was noticing yesterday that baselines does not include the kl divergence in their loss at all.

**Comment by yangysc** [comment]
Yes, they do not add the kl divergence penalty in the loss, but they use an early stopping strategy to prevent large divergence.

https://github.com/DLR-RM/stable-baselines3/blob/740d61ada3360306389210a4dd8ee88ad75592db/stable_baselines3/ppo/ppo.py#L248-L256

**Comment by gjoliver** [comment]
interesting. will sync with sven. thanks for the valuable report.

**Comment by vakker** [comment]
@avnishn sorry to ping on an old issue, but what was the solution to this? The `kl_coeff` divergence is still happening without a warning, and the early stopping that's implemented in SB3 is not part of RLlib.
